packages/block-editor/src/components/block-heading-level-dropdown/index.js

```diff
@@ -28,19 +28,26 @@ const POPOVER_PROPS = {
  */
 
 /**
- * Dropdown for selecting a heading level (1 through 6).
+ * Dropdown for selecting a heading level (1 through 6) or paragraph (0).
  *
  * @param {WPHeadingLevelDropdownProps} props Component props.
  *
  * @return {WPComponent} The toolbar.
  */
-export default function HeadingLevelDropdown( { selectedLevel, onChange } ) {
+export default function HeadingLevelDropdown( {
+	levels,
+	selectedLevel,
```

Comment:
What do you think about using `options` / `value` here? This way, we could match other components.

You can also assign default values here and remove undefined check blow - `levels = HEADING_LEVELS`.